Makefile.toml

```diff
@@ -0,0 +1,212 @@
+    { path = ".env/local.env", profile = "development" },
+]
+
+[config]
+default_to_workspace = false
+skip_core_tasks = true
+
+[env]
+CARGO = "cargo"
+ENGINE_CARGO_TARGET = "wasm32-unknown-unknown"
+SWEEP_DAYS = 30
+
+[tasks.sweep]
+category = "Cleanup"
+install_crate = "cargo-sweep"
+command = "${CARGO}"
+args = [
+    "sweep",
+    "--time",
+    "${SWEEP_DAYS}",
+]
+
+[tasks.clean-cargo]
+category = "Cleanup"
+command = "${CARGO}"
+args = ["clean"]
+
+[tasks.clean-contracts]
+category = "Cleanup"
+script = '''
+rm -Rf etc/eth-contracts/.pnp
+rm -Rf etc/eth-contracts/artifacts
+rm -Rf etc/eth-contracts/cache
+rm -Rf etc/benchmark-contract/target
+'''
+
+[tasks.clean-bin]
+category = "Cleanup"
+script = '''
+rm -Rf bin
+'''
+
+[tasks.clean-flow]
+category = "Cleanup"
+dependencies = [
+    "clean-cargo",
+    "clean-contracts",
+    "clean-bin",
+]
+
+[tasks.clean]
+run_task = "clean-flow"
+
+[tasks.fmt]
+category = "Formatting"
+command = "${CARGO}"
+args = [
+    "fmt",
+    "--all",
+]
+
+[tasks.clippy]
+category = "Check"
+command = "${CARGO}"
+args = [
+    "clippy",
+    "--no-default-features",
+    "--features",
+    "contract",
+    "--",
+    "-D",
+    "warnings",
+]
+
+[tasks.check-fmt]
+category = "Check"
+command = "${CARGO}"
+args = [
+    "fmt",
+    "--",
+    "--check",
+]
+
+[tasks.check-contracts]
+category = "Check"
+script = '''
+cd etc/eth-contracts
+yarn
+yarn lint
+'''
+
+[tasks.check]
+category = "Check"
+dependencies = [
+    "check-contracts",
+    "check-fmt",
+    "clippy",
+]
+
+[tasks.post-engine-build-env]
+category = "Tools"
+script = '''
+echo "Environment:"
+echo "    CARGO_FEATURES:       ${CARGO_FEATURES}"
+echo "    WASM_FILE:            ${WASM_FILE}"
+echo "    TARGET_DIR:           ${TARGET_DIR}"
+echo "    RUSTFLAGS:            ${RUSTFLAGS}"
+echo "    Extra build args:     ${RELEASE} ${@}"
+'''
+
+[tasks.copy-engine-build]
+category = "Post"
+command = "cp"
+args = [
+    "target/wasm32-unknown-unknown/${TARGET_DIR}/aurora_engine.wasm",
+    "bin/${WASM_FILE}",
+]
+
+[tasks.make-bin-directory]
+category = "Post"
+command = "mkdir"
+args = [
+    "-p",
+    "bin",
+]
+
+[tasks.build-contracts]
+category = "Build"
+script = '''
+cd etc/eth-contracts
+yarn
+yarn build
+'''
+
+[tasks.build-engine]
+category = "Build"
+command = "${CARGO}"
+args = [
+    "build",
+    "--target",
+    "${ENGINE_CARGO_TARGET}",
+    "@@remove-empty(RELEASE)",
+    "--no-default-features",
+    "--features=${CARGO_FEATURES}",
+    "-p",
+    "aurora-engine",
+    "-Z",
+    "avoid-dev-deps",
+    "${@}",
+]
+
+[tasks.build-engine-flow]
+category = "Build"
+dependencies = [
+    "build-contracts",
+    "build-engine",
+    "make-bin-directory",
+    "copy-engine-build",
+    "post-engine-build-env",
+]
+
+[tasks.build-test]
+env = { "RUSTFLAGS" = "${RUSTC_FLAGS_BUILD}", "CARGO_FEATURES" = "${CARGO_FEATURES_BUILD_TEST}", "WASM_FILE" = "${WASM_FILE_TEST}", "RELEASE" = "--release", "TARGET_DIR" = "release" }
```

Comment:
I ran `cargo make build-test` command after cloning and installed prerequisite, and it seems `Makefile.toml` cannot get `RUSTC_FLAGS_BUILD` environment variable.

```
[cargo-make] INFO - Execute Command: "cargo" "build" "--target" "wasm32-unknown-unknown" "--release" "--no-default-features" "--features=testnet" "-p" "aurora-engine" "-Z" "avoid-dev-deps"
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names '${RUSTC_FLAGS_BUILD}' --target wasm32-unknown-unknown --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit status: 1)
  --- stderr
  error: multiple input filenames provided (first two filenames are `-` and `${RUSTC_FLAGS_BUILD}`)
```

After I added `RUSTC_FLAGS_BUILD` as `[env]` variable in the file, the command works. Is there something that I missed in prerequisite?

Reply:
Good catch, will fix.